docs/user-guide/01-blueprint-reference.md

```diff
@@ -1296,48 +1296,37 @@ minsize = "1 GiB"
 ```
 </TabItem>
 <TabItem value="bootc" >
-```json
-{
-  "customization": {
-    "disk": {
-      "partitions": [
-        {
-          "type": "plain",
-          "label": "data",
-          "mountpoint": "/data",
-          "fs_type": "ext4",
-          "minsize": "50 GiB"
-        },
-        {
-          "type": "lvm",
-          "name": "mainvg",
-          "minsize": "20 GiB",
-          "logical_volumes": [
-            {
-              "name": "rootlv",
-              "mountpoint": "/",
-              "label": "root",
-              "fs_type": "ext4",
-              "minsize": "2 GiB"
-            },
-            {
-              "name": "homelv",
-              "mountpoint": "/home",
-              "label": "home",
-              "fs_type": "ext4",
-              "minsize": "2 GiB"
-            },
-            {
-              "name": "swaplv",
-              "fs_type": "swap",
-              "minsize": "1 GiB"
-            }
-          ]
-        }
-      ]
-    }
-  }
-}
+```toml
```

Comment:
I wrote the bootc examples as json here because bootc supports both and I thought it would be good to have an example of what it looks like in json (since the on-prem examples use toml anyway).

Maybe we could add a new tab in our examples here so we could have `bootc (toml)` and `bootc (json)`, with an explanation at the top that both are supported in bootc.

Reply:
@ondrejbudai suggested to stick to `toml` (or at least one format, for the whole documentation) for now - I'm fine with adding both if this is the way forward.
As it supports both, it makes sense to document both, I agree…